Formula/mdds.rb

```diff
@@ -1,6 +1,6 @@
 class Mdds < Formula
   desc "Multi-dimensional data structure and indexing algorithm"
-  homepage "https://gitlab.com/mdds/mdds"
+  homepage "https://gitlab.com/mdds/mdds.git"
```

Comment:
Shouldn't this be HEAD instead of homepage?

Reply:
Oh yeah, I'll change that.